subsys/mgmt/Kconfig

```diff
@@ -57,18 +56,60 @@ config MCUMGR_SMP_UART_MTU
 
 endif
 
+config MCUMGR_SMP_UDP
+	bool "UDP mcumgr SMP transport"
+	select MCUMGR
+	select NETWORKING
+	select NET_IPV4
```

Comment:
Why not IPv6 support? From application point of view having IPv6 support is trivial. We are trying to make the IPv6 as a 1st class citizen and this is not helping.